RecoLocalTracker/SiPixelRecHits/plugins/BuildFile.xml

```diff
@@ -1,18 +1,28 @@
 <use name="fmt"/>
-<use name="CUDADataFormats/BeamSpot"/>
-<use name="CUDADataFormats/TrackingRecHit"/>
-<use name="HeterogeneousCore/CUDACore"/>
+<use name="CondFormats/SiPixelTransient"/>
+<use name="DataFormats/BeamSpot"/>
+<use name="DataFormats/TrackerCommon"/>
+<use name="DataFormats/TrackingRecHitSoA"/>
+<use name="HeterogeneousCore/AlpakaCore"/>
+<use name="HeterogeneousCore/AlpakaInterface"/>
 <use name="RecoLocalTracker/ClusterParameterEstimator"/>
 <use name="RecoLocalTracker/Records"/>
 <use name="RecoLocalTracker/SiPixelRecHits"/>
-<use name="DataFormats/TrackerCommon"/>
-<use name="CondFormats/SiPixelTransient"/>
 <iftool name="cuda-gcc-support">
   <use name="cuda"/>
```

Comment:
I'd think the `cuda` dependence here would get applied also to `RecoLocalTrackerSiPixelRecHitsPluginsPortable`, so it should be moved under the `RecoLocalTrackerSiPixelRecHitsPlugins` definition too (inside an `iftool` element).